### PR TITLE
RDoc-1800-backup-config-options

### PR DESCRIPTION
**Pull request by morhilai**
-`Backup.LowMemoryBackupDelayInMin` was already documented
-Documented `Server.CpuCredits.ExhaustionBackupDelayInMin`

### CODE REVIEW

File: Documentation/4.2/Raven.Documentation.Pages/server/configuration/backup-configuration.markdown [hunk 1]
```diff
@@ -13,6 +13,7 @@
     * [Backup.MaxNumberOfConcurrentBackups](../../server/configuration/backup-configuration#backup.maxnumberofconcurrentbackups)  
     * [Backup.ConcurrentBackupsDelayInSec](../../server/configuration/backup-configuration#backup.concurrentbackupsdelayinsec)  
     * [Backup.LowMemoryBackupDelayInMin](../../server/configuration/backup-configuration#backup.lowmemorybackupdelayinmin)  
+    * [Server.CpuCredits.ExhaustionBackupDelayInMin](../../server/configuration/backup-configuration#server.cpucredits.exhaustionbackupdelayinmin)
```
Comment (ppekrol) [inline Documentation/4.2/Raven.Documentation.Pages/server/configuration/backup-configuration.markdown:16]: Let's keep the same layout as in RavenDB. This should be in server-configuration.markdown

Are we missing that file? Is it synced with... 
4.2: https://github.com/ravendb/ravendb/blob/v4.2/src/Raven.Server/Config/Categories/ServerConfiguration.cs
5.0: https://github.com/ravendb/ravendb/blob/v5.0/src/Raven.Server/Config/Categories/ServerConfiguration.cs
5.1: https://github.com/ravendb/ravendb/blob/v5.1/src/Raven.Server/Config/Categories/ServerConfiguration.cs

?

Comment (morhilai) [reply Documentation/4.2/Raven.Documentation.Pages/server/configuration/backup-configuration.markdown]: Fixed.
We weren't missing the file, I just thought the config option was more relevant to backups than to the other server options. So I've moved it to server-configuration, and linked to it from the backup-config page.

File: Documentation/4.2/Raven.Documentation.Pages/server/configuration/backup-configuration.markdown [hunk 2]
```diff
@@ -90,6 +92,10 @@ Number of minutes to delay the backup if the server enters a low memory state.
 
 {PANEL/}
 
+{WARNING: Advanced configuration options - experts only}{WARNING/}
```
Comment (ppekrol) [inline Documentation/4.2/Raven.Documentation.Pages/server/configuration/backup-configuration.markdown:95]: We should remove this?

Comment (morhilai) [reply Documentation/4.2/Raven.Documentation.Pages/server/configuration/backup-configuration.markdown]: Oops :/ fixed 👍

**Pull request merged**